packages/storybook/src/generators/cypress-project/cypress-project.ts

```diff
@@ -30,31 +28,61 @@ export interface CypressConfigureSchema {
   standaloneConfig?: boolean;
   ciTargetName?: string;
   skipFormat?: boolean;
+  projectNameAndRootFormat?: 'as-provided' | 'derived';
 }
 
 export async function cypressProjectGenerator(
   tree: Tree,
   schema: CypressConfigureSchema
 ) {
+  return await cypressProjectGeneratorInternal(tree, {
+    projectNameAndRootFormat: 'derived',
```

Comment:
we're keeping this hard-coded then?

Reply:
Yeah, this is going away in v18 anyway, there's no point to change it.